src/coffee/client/client.coffee

```diff
@@ -3,22 +3,36 @@
 # All rights reserved.
 #
 
-angular = require 'angular'
-
-############################################################################################################
-
 # Allow Node.js-style `global` in addition to `window`
 if typeof(global) is 'undefined'
     window.global = window
 
+# Load JSData libraries. The core library must be first.
+require 'js-data'
+window.DSLocalStorageAdapter = require 'js-data-localstorage'
+require 'js-data-angular'
+
+# Include the Reflux module
+require './modules/reflux'
+
 # Add 'require' statements for your other Angular module files here.
-require './modules/sample_module'
+require './modules/client_schema'
+require './modules/todo/todo_item_module'
+require './modules/todo/todo_list_module'
+require './modules/todo/input_module'
+require './../read_only_view'
```

Comment:
Minor point, but the `ReadOnlyView` is a client-side thing, and therefore could have been included in the `client` directory.